packages/terra-site/package.json

```diff
@@ -42,7 +42,7 @@
     "terra-application": "0.3.0",
     "terra-arrange": "^0.1.0",
     "terra-button": "^0.1.0",
-    "terra-legacy-theme": "^0.1.0",
+    "terra-legacy-theme": "0.3.0",
```

Comment:
Is there a reason we're locking this down but not locking any of the other terra components down?